<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserWindow</class>
 <widget class="QMainWindow" name="UserWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>680</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>680</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>城市GDP信息系统</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/image/icon.png</normaloff>:/image/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">
</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1,1">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_windowname">
         <property name="font">
          <font>
           <family>华文行楷</family>
           <pointsize>20</pointsize>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>城市 GDP 信息系统</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_2" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="maximumSize">
             <size>
              <width>40</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 0);</string>
            </property>
            <property name="text">
             <string>用户名：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_username">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>20</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>退出</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_3" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="pushButton_add">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>增加</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_update">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>修改</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_delete">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>删除</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_clear">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>清空</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_predict">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>预测</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="minimum">
          <number>11</number>
         </property>
         <property name="maximum">
          <number>44</number>
         </property>
         <property name="value">
          <number>11</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-image: url(:/image/search.png);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_cityname">
         <property name="placeholderText">
          <string>城市名</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_year">
         <property name="placeholderText">
          <string>年份</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_search">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>25</height>
          </size>
         </property>
         <property name="tabletTracking">
          <bool>false</bool>
         </property>
         <property name="focusPolicy">
          <enum>Qt::StrongFocus</enum>
         </property>
         <property name="text">
          <string>查找</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="tableWidget">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
      </property>
      <column>
       <property name="text">
        <string>城市名</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>2021下</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>2021上</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>2018-2020最低</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>2018-2020最高</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>2018-2020平均</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>2018-2020增速</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>2020上</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>2019下</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>2019上</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>2018下</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>2018上</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>2015-2017最低</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>2015-2017最高</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>2015-2017平均</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>2015-2017增速</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>2017下</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>2017上</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>2016下</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>2016上</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>2015下</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>2015上</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>2012-2014最低</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>2012-2014最高</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>2012-2014平均</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>2012-2014增速</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>2014下</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>2014上</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>2013下</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>2013上</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>2012下</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>2012上</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>2009-2011最低</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>2009-2011最高</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>2009-2011平均</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>2009-2011增速</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>2011下</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>2011上</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>2010下</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>2010上</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>2009下</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>2009上</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>2006-2008最低</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>2006-2008最高</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>2006-2008平均</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>2006-2008增速</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>2008下</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>2008上</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>2007下</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>2007上</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>2006下</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>2006上</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>2003-2005最低</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>2003-2005最高</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>2003-2005平均</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>2003-2005增速</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>2005下</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>2005上</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>2004下</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>2004上</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>2003下</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>2003上</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>2000-2002最低</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>2000-2002最高</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>2000-2002平均</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>2000-2002增速</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>2002下</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>2002上</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>2001下</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>2001上</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>2000下</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>2000上</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
